src/Connectors/test/Connector.EFCore.Test/MySqlDbContextOptionsExtensionsTest.cs

```diff
@@ -80,9 +78,13 @@ public void UseMySql_ThrowsIfServiceNameNull()
         Assert.Contains(nameof(serviceName), ex4.Message);
     }
 
-    [Fact]
-    public void AddDbContext_NoVCAPs_AddsDbContext_WithMySqlConnection()
+    [Theory]
+    [InlineData(new[]{"Pomelo.EntityFrameworkCore.MySql"}, typeof(MySqlConnector.MySqlConnection))]
+    [InlineData(new[]{"MySql.EntityFrameworkCore", "MySql.Data.EntityFrameworkCore"}, typeof(MySqlDataConnectionAlias))]
+    public void AddDbContext_NoVCAPs_AddsDbContext_WithMySqlConnection(string[] efAssemblies, Type mySqlConnection)
     {
+        SetTypeLocators(efAssemblies, mySqlConnection);
```

Comment:
When this test fails, it influences the outcome of other tests because it fails to reset. To fix that, at the minimum it should be wrapped in a try-catch, but there's a more elegant solution:
```c#
private sealed class AlternateTypeLocatorScope : IDisposable
{
    private readonly string[] _mySqlEntityAssembliesBackup;
    private readonly string[] _mySqlAssembliesBackup;

    public AlternateTypeLocatorScope(string[] mySqlEntityAssemblies, Type mySqlConnectionType)
    {
        _mySqlEntityAssembliesBackup = EntityFrameworkCoreTypeLocator.MySqlEntityAssemblies;
        EntityFrameworkCoreTypeLocator.MySqlEntityAssemblies = mySqlEntityAssemblies;

        _mySqlAssembliesBackup = MySqlTypeLocator.Assemblies;
        MySqlTypeLocator.Assemblies = new[] { mySqlConnectionType.Assembly.FullName?.Split(',')[0] };
    }

    public void Dispose()
    {
        EntityFrameworkCoreTypeLocator.MySqlEntityAssemblies = _mySqlEntityAssembliesBackup;
        MySqlTypeLocator.Assemblies = _mySqlAssembliesBackup;
    }
}
```
Which enables the tests to use:
```c#
using var scope = new AlternateTypeLocatorScope(efAssemblies, mySqlConnection);
```